src/content/docs/apm/agents/java-agent/frameworks/scala-zio.mdx

```diff
@@ -50,44 +50,50 @@ To access the API class add the following information to your Scala configuratio
 
 </CollapserGroup>
 
-Supported Scala versions are 2.11, 2.12 and 2.13. Scala 3.0 users can use the 2.13 jar. 
-The minimum scala version for ZIO v2 is 2.12.
+Supported Scala versions are 2.11, 2.12, and 2.13. Scala 3.0 users can use the 2.13 jar. 
+The minimum Scala version for ZIO v2 is 2.12.
 
-ZIO v1 is supported since agent version 7.2.0 and ZIO v2 is supported since the agent version 8.10.0.
+ZIO v1 is supported since Java agent version 7.2.0 and ZIO v2 is supported since the agent version 8.10.0.
```

Comment:
@obenkenobi could we delete this line since it seems to be saying the same thing as the two bullets in the callout below?